documentation/source/Policies/storage_policy.rst

```diff
@@ -0,0 +1,127 @@
+
+Storage Policies
+==================
+You can now define policies based on the type of application at the storage-class level. Following are some of the properties that can be customized at the default level in the openebs-storageclasses.yaml file.
+
+* jiva-replica-count
+* jiva-replica-image
+* jiva-controller-image
+* volume capacity and monitoring 
```

Comment:
let us split this into separate lines

Reply:
storage pool policy is missing

Reply:
Added

Reply:
This is a bullet list. Separate lines meaning?